projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -1,130 +1,141 @@
+        <VDKTextInput
+          option={VdkOption.REST_API_URL}
+          value="http://my_vdk_instance"
+          label="Rest API URL:"
+        ></VDKTextInput>
+        <VDKTextInput
+          option={VdkOption.PATH}
+          value={this.props.jobPath}
+          label="Path to job directory:"
+        ></VDKTextInput>
+        <VDKTextInput
+          option={VdkOption.DEPLOYMENT_REASON}
+          value="reason"
+          label="Deployment reason:"
+        ></VDKTextInput>
+        <div>
+          <input
+            type="checkbox"
+            name="enable"
+            id="enable"
+            className="jp-vdk-checkbox"
+            onClick={this.onEnableClick()}
+          />
+          <label className="checkboxLabel" htmlFor="enable">
+            Enable
+          </label>
+        </div>
+      </>
+    );
+  }
+  /**
+   * Callback invoked upon choosing Enable checkbox
    */
-    private onPathChange = (event: any): void => {
-        this.populateJobData(VdkOption.PATH, event.currentTarget as HTMLInputElement);
-    };
-    /**
-  * Callback invoked upon changing the deployent reason input.
-  *
-  * @param event - event object
-  */
-    private onDeploymentReasonChange = (event: any): void => {
-        this.populateJobData(VdkOption.DEPLOYMENT_REASON, event.currentTarget as HTMLInputElement);
+  private onEnableClick() {
+    return (event: React.MouseEvent) => {
+      let checkbox = document.getElementById('enable');
+      if (checkbox?.classList.contains('checked')) {
+        checkbox.classList.remove('checked');
+        jobData.set(VdkOption.DEPLOY_ENABLE, '');
+      } else {
+        checkbox?.classList.add('checked');
+        jobData.set(VdkOption.DEPLOY_ENABLE, '1');
+      }
     };
-    /**
-  *  Helper function for populating jobData
-  */
-    private populateJobData(option: VdkOption, input: HTMLInputElement) {
-        let value = input.value;
-        if (!value) value = "";
-        jobData.set(option, value);
+  }
+}
+
+export async function showCreateDeploymentDialog() {
+  const result = await showDialog({
+    title: 'Create Deployment',
+    body: (
+      <DeployJobDialog
+        jobName="job-to-deploy"
+        jobPath={sessionStorage.getItem('current-path')!}
+        jobTeam="my-team"
+      ></DeployJobDialog>
+    ),
+    buttons: [Dialog.okButton(), Dialog.cancelButton()]
+  });
+  const resultButtonClicked = !result.value && result.button.accept;
+  if (resultButtonClicked) {
+    try {
+      const runConfirmationResult = await showDialog({
+        title: 'Create deployment',
+        body: 'The job will be executed once before deployment.\n Check the result and decide whether you will continue with the deployment, please!',
+        buttons: [
+          Dialog.cancelButton({ label: 'Cancel' }),
+          Dialog.okButton({ label: 'Continue' })
+        ]
+      });
+      if (runConfirmationResult.button.accept) {
+        let success = await jobRunRequest();
```

Comment:
This isn't very clear. Please try destructuring and then checking the values, e.g.

```
let { message, status } = await jobRunRequest();
```

Reply:
done